subsys/bluetooth/controller/ll_sw/ull_filter.c

```diff
@@ -588,6 +588,11 @@ void ull_filter_rpa_update(bool timeout)
 	}
 }
 
+void ull_filter_rpa_refresh(void)
+{
+	k_work_schedule(&rpa_work, K_NO_WAIT);
```

Comment:
I have already suggested to remove this, but you probably meant to have reschedule here?

Reply:
yes, it should be reschedule. The documentation for k_work has so many redirections as to the meaning of parameters to pass, its very easy to pick the wrong API.

Reply:
Yes, k_work is a bit more complicated now, so maybe just leave it. Using k_work is simplest when all you do is just having it at a periodic interval, which is the only requirement the RPA timer has.